webapp/src/pages/viewprofile/components/ViewProfileComponent.js

```diff
@@ -0,0 +1,295 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router';
+import { profileService} from "../../../services/profilelist";
+import { createColClassName } from "../../../utils/styling/styling";
+import { Form } from 'react-bootstrap';
+
+const DEFAULT_EVENT_ID = process.env.REACT_APP_DEFAULT_EVENT_ID || 1;
+ 
+class ViewProfileComponent extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            user: {},
+            loading: true,
+            error:"",
+            isNull:true
+        }
+    }
+
+isUserNull(user){
```

Comment:
This is nit, but is this method necessary? I think it's clearer to just "inline" this and have have `result.data === null`  in your setState function directly.

Reply:
dano....